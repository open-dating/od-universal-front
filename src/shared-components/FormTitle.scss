.form-title {
  margin: 20px 0 0 0;

  small {
    color: rgba(0, 0, 0, 0.54);
  }
}
