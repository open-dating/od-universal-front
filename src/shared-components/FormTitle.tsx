import React from 'react'

import './FormTitle.scss'

export function FormTitle({title, subTitle = ''}: {title: string, subTitle?: string}) {
  return (
    <div className="form-title">
      <div>
        {title}
      </div>
      <small>
        {subTitle}
      </small>
    </div>
  )
}
