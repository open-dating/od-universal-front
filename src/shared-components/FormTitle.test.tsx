import React from 'react'
import { shallow } from 'enzyme'
import { FormTitle } from './FormTitle'

describe('FormTitle', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <FormTitle title='foo' subTitle='bar'></FormTitle>,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
