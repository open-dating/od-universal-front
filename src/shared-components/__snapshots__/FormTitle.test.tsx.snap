// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormTitle renders correctly 1`] = `
<div
  className="form-title"
>
  <div>
    foo
  </div>
  <small>
    bar
  </small>
</div>
`;
